.github/workflows/sync_issue_to_jira.yaml

```diff
@@ -9,9 +9,13 @@ on:
 jobs:
   sync:
     name: Sync GitHub issue to Jira
-    uses: "canonical/data-platform-workflows/\
-      .github/workflows/sync_issue_to_jira.yaml@v1"
+    uses: canonical/data-platform-workflows/.github/workflows/sync_issue_to_jira.yaml@v2
     with:
-      jira-component-name: mysql-k8s
+      jira-base-url: https://warthogs.atlassian.net
+      jira-project-key: DPE
+      jira-component-names: mysql-k8s,mysql-router-k8s,rock
```

Comment:
@taurus-forever 
not related to the PR, but should we split the rocks at jira ?